pkg/reconciler/persistentvolumeclaim/resources.go

```diff
@@ -159,6 +159,10 @@ func getExpectedPVCsFromCluster(cluster *apiv1.Cluster, instanceName string) []e
 		roles = append(roles, utils.PVCRolePgWal)
 	}
 
+	if cluster.ShouldCreateTablespaces() {
+		roles = append(roles, utils.PVCRolePgTablespace)
+	}
+
```

Comment:
I think this is not necessary. The function buildExpectedPVCs below, does the work...

Reply:
Confirmed. The unit test below works without this. Deleting.
As far as I know, Marco has feedback about this, and I do think we will need to restructure.